cmd/kubeadm/app/apis/kubeadm/defaults.go

```diff
@@ -20,4 +20,6 @@ const (
 	DefaultServiceDNSDomain  = "cluster.local"
 	DefaultServicesSubnet    = "10.12.0.0/12"
 	DefaultKubernetesVersion = "v1.4.1"
+	DefaultAPIBindPort       = 6443
```

Comment:
I think the vast majority of automations use 443. Can you point to an automation in one of the k8s repos that doesn't?